### ISSUE TITLE
Add a Simpl logo to enable rich open graph posts

### ISSUE THREAD

**Issue by jefftriplett** [simplworld/simpl.world.website#23]
We need an image to enable open graph tags for Twitter (works on FB too).

From the Twitter [docs](https://dev.twitter.com/cards/types/summary-large-image):

>  A URL to a unique image representing the content of the page. You should not use a generic image such as your website logo, author photo, or other image that spans multiple pages. Images for this Card support an aspect ratio of 2:1 with minimum dimensions of 300x157 or maximum of 4096x4096 pixels. Images must be less than 5MB in size. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

Here's how we set this for open graph tags in our `_config.yml` file:

```yaml
# Frontmatter fallback settings
# This is the default logo that people see on blog posts
defaults:
  - scope:
      path: ""
    values:
      image: /assets/images/default-card.png
```

## References:

- https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/advanced-usage.md#customizing-image-output

**Comment by jefftriplett** [comment]
We also need this for the favicon.

**Comment by jefftriplett** [comment]
Fixed(ish) in #76. @joelee484 / y'all probably want @chadwhitman to build a better image for you. I gave it my best 3 minutes in Pixelmator though.

**Comment by jefftriplett** [comment]
When a non-designer is given keys to pixel kingdom: ![](https://simple.world/assets/img/default-card.png)
